### DISCUSSION TITLE
在 cmd 當 DJ

### DISCUSSION THREAD

**Discussion by eason8479** [1101-GS4719-C/Python-Community-GS4719-C#853] [優秀專案]
https://github.com/dpayne/cli-visualizer

這是最近發現的一個cmd小工具
緣由是我邊寫功課，邊聽歌時，就在想可不可以更 HIGH 一點
上網亂翻之後就翻到了這個(剛好我有在用 linux)
於是就花了一些時間設定

最後在無數的微調下，我終於能在 cmd 裡面 HIGH 起來了
等等... 我是不是忘了什麼?
歐... 對了我的功課

(以上純為虛構故事，沒有任何作業在演出中受到不當對待)

**Comment by dabbie109303501** [comment]
這也太酷了！但我沒有linux不能試試看 好可惜

**Comment by bonny123865** [comment]
是說 寫功課要多嗨啦
不過那個呈現的東東真的好酷

**Comment by Kotori0501** [comment]
好好笑
感覺好酷喔 
下次讀書讀到一半我也想試試看

**Comment by Chiway1229** [comment]
哇 這個讓我想到小時候電腦放音樂都能有這種幻燈片特效
說不定下次能試試 哈哈

**Comment by williamyen0623** [comment]
好好笑 一個被大家認為最呆版無聊的東西 竟然嗨起來了哈哈

**Comment by ruthchen1020** [comment]
太酷了!!!之前就有看過有人分享類似的酷東西，之後也要試試看!(我可能需要找win的做法...

**Comment by evanchou1204** [comment]
好嗨 這樣以後打開cmd調設定 發現他一直默默的隨著音樂跳來跳去 好有趣

**Comment by Evan0979** [comment]
真的假的感覺好酷超high的哈哈哈

**Comment by StevenWanglolz** [comment]
這樣以後在宿舍就可以嗨起來了哈哈哈

**Comment by CPCpyt** [comment]
好酷！沒想到還有這樣的運用，有被震撼到！
